src/cata_tiles.cpp

```diff
@@ -1899,6 +1900,23 @@ bool cata_tiles::draw_from_id_string( std::string id, TILE_CATEGORY category,
         c ^= b;
         c -= rot32( b, 24 );
         loc_rand = c;
+
+        // idle tile animations:
+        if ( display_tile.animated ) {
```

Comment:
AStyle - should be:

    if( display_tile.animated ) {

Reply:
We have astyle check with a [display corrections step.](https://github.com/CleverRaven/Cataclysm-DDA/pull/40342/checks?check_run_id=655422443#step:5:763)